libs/ebpfnetsh/elf.cpp

```diff
@@ -329,7 +329,7 @@ handle_ebpf_show_verification(
         filename.c_str(), section.c_str(), &program_type, level == VL_VERBOSE, &report, &error_message, &stats);
     if (status == ERROR_SUCCESS) {
         std::cout << report;
-        std::cout << "\nProgram terminates within " << stats.max_instruction_count << " instructions\n";
+        std::cout << "\nProgram terminates within " << stats.max_loop_count << " loop iterations\n";
```

Comment:
question: Ideally should such a change result in creating a new `v2` API? (to avoid any possible regressions?)